python/lsst/analysis/drp/scatterPlot.py

```diff
@@ -57,6 +57,35 @@ class ScatterPlotWithTwoHistsTaskConfig(pipeBase.PipelineTaskConfig,
         itemtype=str
     )
 
+    def get_requirements(self):
```

Comment:
This should have an underscore in the beginning, since this is not meant to be a public API (or is it?). I don't have a good alternative, but `requirements` seems strange. `_get_bands_and_columns`, may be?

Reply:
Sure, it doesn't change state, but it shows up as a method of the `config` in autocomplete or when used with `dir()`. If the user will never need to call this method, then best making it private. But up to you.